src/components/Article.jsx

```diff
@@ -1 +1,23 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, date, text }) => (
```

Comment:
```suggestion
export const Article = ({ 
  title, 
  date, 
  text 
}) => (
```

if we have three or more values, it is better to start them from new line

Reply:
✔️